src/app/shared/components/header/header.component.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../../../styles/base/paths";
+
+.header {
+  overflow: hidden;
+  background-color: #121B23;
+  padding: 20px 10px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header a {
+  float: left;
+  color: #8BCDEA;
+  text-align: center;
+  padding: 12px;
+  text-decoration: none;
+  font-size: 18px;
+  line-height: 25px;
+  border-radius: 4px;
+  cursor: pointer;
+  margin-right: 30px;
+}
```

Comment:
this block to be moved inside the upper block => to be 
.header {
...
a {
...
}
}